tensorflow/docs_src/get_started/get_started.md

```diff
@@ -29,7 +29,9 @@ working internally when you use the more compact higher level API.
 
 # Tensors
 
-The central unit of data in TensorFlow is the **tensor**. A tensor consists of a
+The central unit of data in TensorFlow is the **tensor**. The formal physicist's and 
```

Comment:
Some physicists can be very formal, but I don't think we need to go into this :-)

Jokes aside, I am not sure this clarification is particularly useful for most users of tensorflow, and I think that introducing more formal concepts like "map" and "basis" here will confuse more users than it will help. I think the users who know what those terms mean will also know that the concrete "tensors" in TensorFlow would be a representation of an abstract tnesor in a particular basis.